workflow/snakemake/input.yaml

```diff
@@ -0,0 +1,6 @@
+data:
+  class: Directory
+  path: data
+code:
+  class: Directory
+  path: code
```

Comment:
What is this file for? Normally we use it to define params, but I don't see any `config[]` reference in the Snakefile.

Reply:
Added the configuration setting to the Snakefile